core/base-service/got.js

```diff
@@ -3,6 +3,8 @@
 const got = require('got')
 const { Inaccessible, InvalidResponse } = require('./errors')
 
+const userAgent = 'Shields.io/2003a'
```

Comment:
I have no idea what this number means. I just copied it from https://github.com/badges/shields/blob/2359eb278beb4612ea8e579b062648463dc79813/core/base-service/legacy-request-handler.js#L14

I decided to just duplicate this in `got.js` instead of importing it from legacy-request-handler as I think we can get rid of a lot of what is in there in the near future

Reply:
That.. sort of makes sense for self-hosted installs (unless they track next/master), although we don't pin our own deploys to a version, by which I mean

we tag server-YYYY-MM-DD and deploy
we make some more commits and deploy those too
now the version we're running in production isn't server-YYYY-MM-DD - its a future version that doesn't correspond to any tag

I suppose `Shields.io/server-YYY-MM-DD` would at least mean `server-YYY-MM-DD` (or later)

Reply:
Sure. We could also use `server-YYY-MM-DD` for the Docker releases and includ something like `server-YYY-MM-DD+dev` for production. 

Reading this thread makes me think we should include some extra stuff in the query string on request for the dynamic + endpoint badges, or when a user-specified URL is used, since it allows a service provider to differentiate between our official badge requests, and other requests which are being specified by the user.

Reply:
I've made an issue for this as I suspect we're not going to pick this up immediately and this conversation will disappear into the github ether. https://github.com/badges/shields/issues/6268